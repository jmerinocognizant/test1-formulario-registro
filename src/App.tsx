import './App.css';
import 'bootstrap-icons/font/bootstrap-icons.css'
import { FormularioRegistro } from './components/FormularioRegistro';

function App() {
  return (
    <FormularioRegistro />
  );
}

export default App;
