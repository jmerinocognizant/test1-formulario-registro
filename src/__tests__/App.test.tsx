import React from 'react';
import { render, screen } from '@testing-library/react';
import App from '../App';


describe('Pruebas en <Appp />', () => {

  test('debe de hacer match con el snapshot', () => {
    const { container } = render(<App />);
    expect( container ).toMatchSnapshot();
  });


  test('debe mostrar componente formulario', () => {
    render(<App />);
    //screen.debug();
    expect(screen.getByText("Registration"));
  });
});
