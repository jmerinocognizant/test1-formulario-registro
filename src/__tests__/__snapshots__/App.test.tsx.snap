// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <Appp /> debe de hacer match con el snapshot 1`] = `
<div>
  <div
    class="container login-container rounded mt-5 justify-content-md-center"
  >
    <h2
      class="d-flex justify-content-center text-secondary"
    >
      Registration
    </h2>
    <hr />
    <div
      class="input-group mt-2"
    >
      <span
        class="input-group-text text-white bg-primary"
      >
        <i
          class="bi bi-envelope-fill fs-5"
        />
      </span>
      <input
        aria-label="email"
        class="form-control"
        id="email"
        name="email"
        placeholder="Email"
        type="email"
        value=""
      />
    </div>
    <hr />
    <div
      class="input-group mt-2"
    >
      <span
        class="input-group-text text-white bg-primary"
      >
        <i
          class="bi bi-person-fill fs-5"
        />
      </span>
      <input
        aria-label="name"
        class="form-control"
        id="name"
        name="name"
        placeholder="Name"
        type="text"
        value=""
      />
    </div>
    <hr />
    <div
      class="input-group mt-2"
    >
      <span
        class="input-group-text text-white bg-primary"
      >
        <i
          class="bi bi-person-fill fs-5"
        />
      </span>
      <input
        class="form-control"
        data-testid="password"
        id="password"
        name="password"
        placeholder="Password"
        type="password"
        value=""
      />
    </div>
    <hr />
    <div
      class="form-check"
    >
      <input
        aria-label="check-privacy-policy"
        class="form-check-input border-primary rounded-0"
        id="privacyPolicy"
        name="privacyPolicy"
        type="checkbox"
        value="privacyPolicy"
      />
      <label
        class="form-check-label text-secondary"
        for="checkbox-privacy"
      >
        I Agree to Privacy Policy
      </label>
    </div>
    <hr />
    <button
      aria-label="btn-submit"
      class="btn btn-primary w-100"
      disabled=""
      type="button"
    >
      Submit
    </button>
  </div>
</div>
`;
